flight/targets/sparky2/fw/board.h

```diff
@@ -0,0 +1,69 @@
+/*
+    ChibiOS/RT - Copyright (C) 2006-20s13 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#ifndef _BOARD_H_
+#define _BOARD_H_
+
+#include "pios_chibios_transition_priv.h"
+
+/*
+ * Setup for STMicroelectronics STM32F4-Discovery board.
+ */
+
+/*
+ * Board identifier.
+ */
+#define BOARD_QUANTEC_QUANTON
+#define BOARD_NAME                  "quanton flight control"
```

Comment:
I can update it. I copied the commit as-is from @peabody124's branch. When I did the ChibiOS/RT upgrade for BrainFPV, it was named similarly, and the comment on line 23 was regarding a Discovery board too. I updated the Brain one.

Preferences? I can easily update!